src/UI/COSetupController.java

```diff
@@ -70,13 +70,16 @@ public boolean handleInput(InputAction action)
         // Build the CO list and the new map and create the game instance.
         Commander[] cos = gameBuilder.commanders.toArray(new Commander[gameBuilder.commanders.size()]);
         GameMap map = new GameMap( cos, gameBuilder.mapInfo );
-        GameInstance newGame = new GameInstance(map, cos);
+        if( map.initOK() )
```

Comment:
Might be worth doing something to actually inform the user why you quit to main menu, but I suppose it's not worth doing anything about it currently.